src/components/MenuBars/MenuBars.tsx

```diff
@@ -110,74 +111,151 @@ export const MenuBars = ({showPreviousColumn, showNextColumn, onPreviousColumn,
+              toggleStartLabel={t("MenuBars.raiseHand")}
+              toggleStopLabel={t("MenuBars.lowerHand")}
+              icon={RaiseHand}
+              onToggle={toggleRaiseHand}
+              value={raisedHand}
             />
+            <MenuButton tabIndex={TabIndex.UserMenu + 2} direction="right" label={t("MenuBars.settings")} onClick={showSettings} icon={SettingsIcon} />
           </div>
-        )}
 
+          <button
+            className={classNames("menu-bars__navigation", {"menu-bars__navigation--visible": showPreviousColumn || showNextColumn})}
+            disabled={!showPreviousColumn}
+            onClick={onPreviousColumn}
+            aria-hidden
+          >
+            <LeftArrowIcon className="menu-bars__navigation-icon" />
+          </button>
+        </section>
+
+        <section className={classNames("menu", "admin-menu", {"admin-menu--empty": !isAdmin, "menu-animation": animate})} onTransitionEnd={(event) => handleAnimate(event)}>
+          {isAdmin && (
+            <div className="menu__items">
+              <MenuButton tabIndex={TabIndex.AdminMenu} direction="left" label="Timer" onClick={showTimerMenu} icon={TimerIcon} />
+              <MenuButton tabIndex={TabIndex.AdminMenu + 1} direction="left" label="Voting" onClick={showVotingMenu} icon={VoteIcon} />
+              <MenuToggle
+                value={state.moderation}
+                direction="left"
+                toggleStartLabel={t("MenuBars.startFocusMode")}
+                toggleStopLabel={t("MenuBars.stopFocusMode")}
+                icon={FocusIcon}
+                onToggle={toggleModeration}
+                tabIndex={TabIndex.AdminMenu + 2}
+              />
+            </div>
+          )}
+
+          <button
+            className={classNames("menu-bars__navigation", {"menu-bars__navigation--empty": !isAdmin, "menu-bars__navigation--visible": showPreviousColumn || showNextColumn})}
+            disabled={!showNextColumn}
+            onClick={onNextColumn}
+            aria-hidden
+          >
+            <RightArrowIcon className="menu-bars__navigation-icon" />
+          </button>
+        </section>
+      </aside>
+      <aside className="menu-bars-mobile">
         <button
-          className={classNames("menu-bars__navigation", {"menu-bars__navigation--empty": !isAdmin, "menu-bars__navigation--visible": showPreviousColumn || showNextColumn})}
-          disabled={!showNextColumn}
-          onClick={onNextColumn}
-          aria-hidden
+          className={classNames("menu-bars-mobile__fab menu-bars-mobile__fab-main", {"menu-bars-mobile__fab-main--isExpanded": fabIsExpanded})}
+          onClick={() => {
+            setFabIsExpanded(!fabIsExpanded);
+          }}
+          aria-label={t("MenuBars.openMenu")}
         >
-          <RightArrowIcon className="menu-bars__navigation-icon" />
+          {fabIsExpanded ? <CloseIcon aria-hidden /> : <MenuIcon className="menu-bars-mobile__fab-main-icon" aria-hidden />}
```

Comment:
<img width="414" alt="Bildschirmfoto 2022-07-26 um 17 23 12" src="https://user-images.githubusercontent.com/35272402/181045921-696a1ac3-80af-4592-a7db-079f171a39b2.png">

On WebKit (all iOS Browsers), the close Icon is the same color as the background and therefore not visible (both light and dark mode)

Reply:
Also, what do you think about the menu closing automatically when pressing outside of it? We have an clickOutside event handler already implemented in `ColumSettings.tsx:37`

Reply:
The icon had the correct color but somehow the width/height `auto` sizing was buggy in WebKit. I set the size manually and now it works :)